### ISSUE TITLE
Application Launch fails with cross walk tutorial

### ISSUE THREAD

**Issue by mark261092** [mapsplugin/cordova-plugin-googlemaps#731]
Hi,

Presently I'm using Cordova version 5.0.0 and Cordova Platform Version: 4.0.0.
I'm using cross walk plugin and have followed the installation steps for crosswalk. However the app launches and the splash screen shows. but then it goes blank. Can you please give me some suggestions.
I've also manually added "version.xml" in "\platforms\android\res\values" as follows.
```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <integer name="google_play_services_version">6171000</integer>
</resources>
```
My Plugin list is :

```
com.red_folder.phonegap.plugin.backgroundservice 2.0.0 "Background Service Plugi
n - Core logic"
com.red_folder.phonegap.plugin.backgroundservice.sample 2.0.0 "Background Servic
e Plugin - Sample Background Service"
com.synconset.imagepicker 1.0.7 "ImagePicker"
cordova-plugin-camera 1.2.0 "Camera"
cordova-plugin-crosswalk-webview 1.4.0 "Crosswalk WebView Engine"
cordova-plugin-device 1.0.1 "Device"
cordova-plugin-file 3.0.0 "File"
cordova-plugin-file-transfer 1.3.0 "File Transfer"
cordova-plugin-geolocation 1.0.1 "Geolocation"
cordova-plugin-network-information 1.0.1 "Network Information"
cordova-plugin-splashscreen 2.1.0 "Splashscreen"
cordova-plugin-whitelist 1.0.0 "Whitelist"
cordova-plugin-x-toast 2.2.1 "Toast"
cordova-sqlite-storage 0.7.12-pre "Cordova sqlite storage plugin"
plugin.google.maps 1.3.4 "phonegap-googlemaps-plugin"
uk.co.workingedge.phonegap.plugin.LaunchNavigator 2.9.2 "Launch Navigator"
```

**Comment by Jerome2606** [comment]
Hi,

Try to add this in config.xml:
```
<preference name="android-minSdkVersion" value="16" />
```

**Comment by mark261092** [comment]
Sorry, but that didn't work.

My issue is the same as [this issue.](https://github.com/mapsplugin/cordova-plugin-googlemaps/issues/638) 

However their solutions aren't working.


my config file looks like this:

```
<?xml version='1.0' encoding='utf-8'?>
<widget id="com.test.test" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
    <name>ProjectManagers</name>
    <description>
        A sample Apache Cordova application that responds to the deviceready event.
    </description>
    <author email="dev@cordova.apache.org" href="http://cordova.io">
        Apache Cordova Team
    </author>
    <content src="index.html" />
    <plugin name="cordova-plugin-whitelist" version="1" />
    <access origin="*" />
    <allow-intent href="http://*/*" />
    <allow-intent href="https://*/*" />
    <allow-intent href="tel:*" />
    <allow-intent href="sms:*" />
    <allow-intent href="mailto:*" />
    <allow-intent href="geo:*" />
    <preference name="AutoHideSplashScreen" value="false" />
    <preference name="FadeSplashScreen" value="true" />
    <preference name="FadeSplashScreenDuration" value="0.3" />
    <platform name="android">
        <allow-intent href="market:*" />
        <preference name="KeyboardDisplayRequiresUserAction" value="false" />
        <preference name="SplashScreenDelay" value="10000" /> <!-- this is max time, my .hide() method should call before -->
        <preference name="SplashMaintainAspectRatio" value="true" />
    </platform>
    <platform name="ios">
        <allow-intent href="itms:*" />
        <allow-intent href="itms-apps:*" />
    </platform>
    <preference name="android-minSdkVersion" value="16" />
    <preference name="xwalkVersion" value="14+" />
    <preference name="xwalkCommandLine" value="--disable-pull-to-refresh-effect" />
    <preference name="xwalkMode" value="embedded" />
    <preference name="xwalkMultipleApk" value="true" />
    
</widget>

```

**Comment by onnoll** [comment]
+1 @mark261092 

I am working with Ionic and Crosswalk 14 and I followed all the steps in the tutorial.

The Map View works fine in isolation but when you try to start your application from a view without map the screen goes blank.

**Comment by mark261092** [comment]
@onnoll :

This is just a suggestion i got from another issue posted here, so can you please try to initialize the map when the application launches itself and hide it soon after as it isn't needed to be shown here. you can use the setClickable as false to do this, or can set the map canvas as display none. Then you can just show the map in the required page when needed. Hope this helps.

However are you able to add any markers, i'm not able to do this for some reason. Any suggestions will be appreciated.

**Comment by onnoll** [comment]
@mark261092 

I think that we should find a better solution than loading the map at start. The most part of applications don't do that, so I think it is a important trouble for making apps using this plugin along with Crosswalk.

Regarding to your problem with markers, I don't have any problem adding makers dynamically following the common way.

**Comment by Hirbod** [comment]
I've fixed the problem in current master 1.3.5 - grab it, it should not make any problems anymore. Npm is not updated, I will push npm as soon as I have fixed some other pending issues

**Comment by mark261092** [comment]
@Hirbod :

I can confirm that this works well. Thank You!!!
